RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/utils/MethodDataExtractor.java

```diff
@@ -1,23 +1,22 @@
-package ru.hse.kirilenko.refactorings.utils;
+package org.jetbrains.research.extractMethodExperiments.utils;
 
 import com.github.javaparser.ast.body.MethodDeclaration;
 import com.github.javaparser.ast.body.Parameter;
 import com.github.javaparser.ast.type.VoidType;
+import org.jetbrains.research.extractMethodExperiments.legacy.OutputUtils;
 
 import java.io.PrintWriter;
 import java.util.List;
 
-import static ru.hse.kirilenko.refactorings.legacy.OutputUtils.printLn;
-
 public class MethodDataExtractor {
     public static void extractParamsCount(MethodDeclaration md, PrintWriter out) {
         List<Parameter> params = md.getParameters();
         if (params != null) {
-            printLn("PARAMS COUNT: " + params.size(), out);
+            OutputUtils.printLn("PARAMS COUNT: " + params.size(), out);
```

Comment:
!

Reply:
What is wrong here? I just reconfigured imports, it is the same function for making results/<repoName>_output.txt